companion/src/simulation/simulatorinterface.cpp

```diff
@@ -100,7 +100,11 @@ void SimulatorLoader::registerSimulators()
   path.truncate(path.lastIndexOf('\\'));
   dir.setPath(path);
 #else
-  dir.setPath(SIMULATOR_LIB_SEARCH_PATH);
+  if (QDir::isAbsolutePath(SIMULATOR_LIB_SEARCH_PATH)) {
+    dir.setPath(SIMULATOR_LIB_SEARCH_PATH);
+  } else {
+    dir.setPath(QCoreApplication::applicationDirPath() + "/" SIMULATOR_LIB_SEARCH_PATH);
+  }
 #endif
   registerSimulators(dir);
```

Comment:
Would this not be better as `registerSimulators(dir.absolutePath());`?

i.e this is the current output
```
[D] SimulatorLoader::registerSimulators():55 - [simulator.interface.loader] Searching for simulators in "/tmp/.mount_Compan6V9ZnK/local/bin" matching pattern ("*-simulator.so")
[D] SimulatorLoader::registerSimulators():83 - [simulator.interface.loader] Found libraries: ("none")
[D] SimulatorLoader::registerSimulators():55 - [simulator.interface.loader] Searching for simulators in "/tmp/.mount_Compan6V9ZnK/local/bin/../lib/companion26" matching pattern ("*-simulator.so")
[D] SimulatorLoader::registerSimulators():60 - [simulator.interface.loader] Trying to register simulator in  "libopentx-nv14-simulator.so"
```
Whereas absolutepath ensures it becomes an, well, absolute path... so it's a clean path now...
```
[D] SimulatorLoader::registerSimulators():55 - [simulator.interface.loader] Searching for simulators in "/tmp/.mount_CompanQ55QXC/local/bin" matching pattern ("*-simulator.so")
[D] SimulatorLoader::registerSimulators():83 - [simulator.interface.loader] Found libraries: ("none")
[D] SimulatorLoader::registerSimulators():55 - [simulator.interface.loader] Searching for simulators in "/tmp/.mount_CompanQ55QXC/local/lib/companion26" matching pattern ("*-simulator.so")
[D] SimulatorLoader::registerSimulators():60 - [simulator.interface.loader] Trying to register simulator in  "libopentx-tx16s-simulator.so"
```

Reply:
I changed it ;-) Let's see how that works.